client/my-sites/upgrades/checkout-thank-you/failed-purchase-details.jsx

```diff
@@ -52,8 +53,8 @@ const FailedPurchaseDetails = ( { failedPurchases, purchases, translate } ) => {
 			</ul>
 			<p>
 				{
-					translate( 'We have filled your account with credits so you can retry the failed purchases. ' +
-						'If the problem persists, please don\'t hesitate to {{a}}contact support{{/a}}!',
+					translate( 'We added credits to your account, so you can try adding these items again. ' +
+						'If the problem persists, please don’t hesitate to {{a}}contact support{{/a}}.',
```

Comment:
This seems like a funky Unicode apostrophe `’`, not the plain, old `'`. Seems like it got into the original copy by Ran somehow.